main.lua

```diff
@@ -1378,8 +1378,13 @@ function loadconfig(nodefaultconfig)
 	else
 		return
 	end
-	
-	s1 = s:split(";")
+
+	if s:find(";\n") then
+		s1 = s:split(";\n")
```

Comment:
some text editors automatically insert newlines at the end of text files which, while unlikely, could erroneously trigger this newline-based split when loading manually-edited config files which could cause the config to fail to load. an easy fix (at least in other langs, idk about lua) would be to strip the string of leading/trailing whitespace

Reply:
It already saves them with a newline at the end (since they're put at the end of every line), and it doesn't cause any issues

Reply:
Oh wait I see what you mean, if it already has a newline at the end from it being edited before they update